runtime/tutor.txt

```diff
@@ -415,6 +422,303 @@ _________________________________________________________________
 
 
 
+=================================================================
+=                   SELECTING TO A CHARACTER                    =
+=================================================================
+
+ Select up to, and including a character with f, followed by that
+ character. Select backwards to a character with F (uppercase).
+ The f stands for 'find'.
+
+ 1. Move the cursor to the line below marked -->. Make sure your
+    cursor is put just after the arrow.
+ 2. press f, followed by the character you want to select
+    to, in this case a square bracket: [.
+ 3. press d to delete your selection, breaking the sentence free
+    of the square brackets
+ 4. go to the very end of the line and repeat again with uppercase
+    F. Remember this time you're selecting to a closing ].
+
+ --> -----[This sentence is surrounded in square brackets, free me!]-----
+
+ Note: Unlike vim, helix doesn't limit this command to just one
+       line. It'll just search for that character in the file.
+
+
+
+
+
+=================================================================
+=                  SELECTING TIL' A CHARACTER                   =
+=================================================================
+
+ Select up to, but NOT including ('til) a character with t. Select backwards
+ until a character with uppercase T
+
+ 1. Move the cursor to the line below marked -->.
+ 2. press t and then A to select until A.
+ 3. press d to delete the dashes.
+ 4. Go to the line end, and repeat with T. Only
+    this time you're searching for an 's'
+
+ --> -----A sentence surrounded in dashes-----
+
+ Note: Unlike vim, helix doesn't limit this command to just one
+       line. It'll just search for that character in the file.
+
+
+
+
+
+=================================================================
+=                             RECAP                             =
+=================================================================
+
+ * Press C to copy the selection on the line below and Alt-C for
+   above.
+
+ * Press s to select all instances of a regex pattern inside
+   the current selection.
+
+ * Press semicolon ( ; ) to collapse
+
+ * Press f / F to extend selection up to & including a character.
+
+ * press t / T to extend selection until a character.
+
+
+
+
+
+=================================================================
+=                   COPYING AND PASTING TEXT                    =
+=================================================================
+
+ Press y to yank (copy) the selection to the clipboard.
```

Comment:
Yanking uses internal registers, not the clipboard.

Reply:
I just omitted the "to the clipboard" part. I'm not sure about mentioning a topic like registers without saying what they are, and that probably deserves it's own section. Let me know if you disagree.